src/Simplex/Messaging/Protocol.hs

```diff
@@ -71,11 +71,13 @@ data Command (a :: Party) where
   OFF :: Command Recipient
   DEL :: Command Recipient
   SEND :: MsgBody -> Command Sender
+  PING :: Command Sender
```

Comment:
not exclusive to sender? well I guess that's just previous SMP abstraction being stretched here.. whatever

but maybe add a separate `Agent` party?

Reply:
also could you separate `Broker` and "Agent" commands with comment headers please? ("Agent" commands are of `Recipient` & `Sender`, `Agent` if you add it as party. `Recipient` and `Sender` could be separate sections)

Reply:
no problem if it is assigned to Sender. We do not create one TCP connection per queue, meaning that in one. TCP connection there can be both "Sender" and "Receiver" commands.

The point to make it belong to Sender is that if/when we add authentication, the server would reject Recipient commands without server user authentication.